recipe/meta.yaml

```diff
@@ -6,24 +6,23 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langsmith-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: a998bbb2514951c6f7718a9bd426917eb95d694d9cfa2bfda54ab348d6c038f1
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - langsmith = langsmith.cli.main:main
 
 requirements:
   host:
-    - python >=3.8,<4.0
```

Comment:
skip: True  # [py<38] ?

Reply:
No need. We build py>=38 by default. Only if there is a difference then we can skip